cita-auth/src/main.rs

```diff
@@ -142,17 +142,20 @@ fn profiler(flag_prof_start: u64, flag_prof_duration: u64) {
 }
 
 fn main() {
-    micro_service_init!("cita-auth", "CITA:auth");
-    info!("Version: {}", get_build_info_str(true));
-
     // init app
     let matches = App::new("auth")
         .version(get_build_info_str(true))
         .long_version(get_build_info_str(false))
         .author("Cryptape")
         .about("CITA Block Chain Node powered by Rust")
         .args_from_usage("-c, --config=[FILE] 'Sets a custom config file'")
+        .args_from_usage("-s, --stdout 'Log to console'")
```

Comment:
```
 .args_from_usage("-c, --config=[FILE] 'Sets a custom config file'
                                -s, --stdout 'Log to console'")
```
```
.args_from_usage("-c, --config=[FILE] 'Sets a custom config file'")
.args_from_usage("-s, --stdout 'Log to console'")
```
If these two style both work well, my advice is choose one.

Reply:
ok, I will amend it.